relay/pkg/btc/remote/remote.go

```diff
@@ -11,19 +16,83 @@ var logger = log.Logger("relay-btc-remote")
 
 // remoteChain represents a remote Bitcoin chain.
 type remoteChain struct {
-	// TODO: implementation
+	rpcClient *rpcclient.Client
+}
+
+func serializeHeader(header *wire.BlockHeader) ([]byte, error) {
+	var buffer bytes.Buffer
+	writer := io.Writer(&buffer)
+	err := header.Serialize(writer)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return buffer.Bytes(), nil
 }
```

Comment:
It's an auxiliary function non-related with the `remoteChain` struct. Can we move it to the bottom? I think it will be good to keep the usual order of types and functions in the file:
1. Main type declaration (`remoteChain`) + auxiliary types 
2. Main type constructor (`Connect` in this case)
3. Main type methods 
4. Auxiliary functions